core/templates/pages/about-page/about-page.component.css

```diff
@@ -313,7 +324,8 @@
     padding-bottom: 5vh;
   }
   .oppia-about-page-section-one .oppia-about-title {
-    padding-top: 10vh;
+    font-size: 25px;
```

Comment:
Why do we need to change the font size...?

Reply:
updated this also. Thank You.